e2etests-cli/main_test.go

```diff
@@ -2,20 +2,42 @@ package e2etests_cli
 
 import (
 	"context"
+	"path/filepath"
 	"testing"
 	"time"
+
+	"oss.terrastruct.com/d2/d2cli"
+	"oss.terrastruct.com/util-go/assert"
+	"oss.terrastruct.com/util-go/xmain"
+	"oss.terrastruct.com/util-go/xos"
 )
 
 func TestCLI_E2E(t *testing.T) {
 	t.Parallel()
 
 	tca := []struct {
 		name string
-		run  func(t *testing.T, ctx context.Context)
+		run  func(t *testing.T, ctx context.Context, dir string, env *xos.Env)
 	}{
 		{
-			name: "hello_world",
-			run:  func(t *testing.T, ctx context.Context) {},
+			name: "hello_world_png",
+			run: func(t *testing.T, ctx context.Context, dir string, env *xos.Env) {
+				writeFile(t, dir, "hello-world.d2", `x -> y`)
+				err := runTestMain(t, ctx, dir, env, "hello-world.d2", "hello-world.png")
+				assert.Success(t, err)
+				png := readFile(t, dir, "hello-world.png")
+				assert.Testdata(t, ".png", png)
+			},
+		},
+		{
+			name: "hello_world_png_pad",
+			run: func(t *testing.T, ctx context.Context, dir string, env *xos.Env) {
+				writeFile(t, dir, "hello-world.d2", `x -> y`)
+				err := runTestMain(t, ctx, dir, env, "-pad 4000", "hello-world.d2", "hello-world.png")
```

Comment:
it's `--pad`

weirdly it throws an error when value too high 

<img width="473" alt="Screen Shot 2023-03-02 at 8 09 38 PM" src="https://user-images.githubusercontent.com/3120367/222629247-0bad8f11-f44c-45fe-8b19-86d93c651192.png">

Reply:
Yup already fixed.

Reply:
That is super weird though.

Reply:
Maybe the PNG gets too big?